ChangeLog.d/use-psa-ecdhe-curve.txt

```diff
@@ -0,0 +1,7 @@
+Bugfix
```

Comment:
Why doesn't this bug fix have a non-regression test?

Reply:
Because it would require the server to send a malformed ServerHello, and I don't think we have the necessary tooling for that.

I'm merging this for now, as this is approved and there's another PR that depends on the upstream PR, but if I'm mistaken and we can test with malformed ServerHello, I'll create a follow-up PR with the non-regression test.